### PR TITLE
Don't reopen everything when we just want to reopen the CDVD

### PR DESCRIPTION
**Pull request by TheLastRar**
This was causing DEV9 code to crash

Possibly also fixes some CDVD issues that where apparently happening?

### CODE REVIEW

**Comment by lightningterror** [comment]
Can you run clang format on the file while you're at it?

**Comment by TheLastRar** [comment]
Done

File: pcsx2/System/SysThreadBase.cpp [hunk 1]
```diff
@@ -199,48 +209,55 @@ void SysThreadBase::PauseSelfDebug()
...
 			Start();
-			if( !m_running || (m_ExecMode == ExecMode_NoThreadYet) )
+			if (!m_running || (m_ExecMode == ExecMode_NoThreadYet))
 				throw Exception::ThreadCreationError(this);
-			if( m_ExecMode == ExecMode_Opened ) return;
+			if (m_ExecMode == ExecMode_Opened)
+				return;
 		}
-		// fall through...
+			// fall through...
```
Comment (refractionpcsx2) [inline pcsx2/System/SysThreadBase.cpp:238]: Can you get rid of the space, dunno why Clang did that. maybe better still replace it with [[fallthrough]];

Comment (TheLastRar) [reply pcsx2/System/SysThreadBase.cpp]: Done

**Review by lightningterror — APPROVED** [review]

**Review by refractionpcsx2 — APPROVED** [review]

**Review by tadanokojin — APPROVED** [review]

**Pull request merged**